app/app.go

```diff
@@ -132,6 +135,18 @@ func Run(ctx context.Context, conf Config) (err error) {
 			return err
 		}
 
+		c := make(chan os.Signal, 1)
+		signal.Notify(c, os.Interrupt, syscall.SIGTERM)
```

Comment:
no need to do this, a defer should suffice

Reply:
nvm that should be gone already